setup.py

```diff
@@ -46,4 +46,9 @@
         "Documentation": "https://github.com/kevinhartman/qiskit-toqm",
         "Source Code": "https://github.com/kevinhartman/qiskit-toqm",
     },
+    entry_points = {
+        'qiskit.transpiler.routing': [
+            'toqm = qiskit_toqm:ToqmSwapPlugin',
```

Comment:
Should this be:
```suggestion
            'toqm = qiskit_toqm.toqm_plugin:ToqmSwapPlugin',
```

Reply:
I suppose it probably should be. I was depending on the `__init__.py`.